samples/boards/argonkey/src/main.c

```diff
@@ -0,0 +1,340 @@
+/*
+ * Copyright (c) 2018 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <misc/printk.h>
+
+#include <board.h>
+#include <gpio.h>
+#include <i2c.h>
+#include <spi.h>
+#include <sensor.h>
+
+/* #define ARGONKEY_TEST_LOG 1 */
+
+#define LED0_GPIO_NAME		LED0_GPIO_CONTROLLER
```

Comment:
Can't we use directly *_GPIO CONTROLLER and *_GPIO_PIN?

Reply:
I think so. I will change it right away.